src/modules/ModuleStaking/index.vue

```diff
@@ -31,6 +32,32 @@ const rawPoolIds = computed(() => {
   return rawPools.value.map((pool) => pool.id)
 })
 
+const stakeAndRewardTokenIds = computed(() => {
+  if (!rawPools.value) return null
+  return Array.from(new Set([
+    ...rawPools.value.map(pool => pool.stakeToken.id),
+    ...rawPools.value.map(pool => pool.rewardToken.id),
+  ]))
```

Comment:
You traverse `rawPools.value` array twice. It'd be more efficient with a single `.reduce()`.

Reply:
Done